_schemaV2.graphql

```diff
@@ -4285,24 +4285,6 @@ type FeaturedLink {
   title: String
 }
 
-type FeatureImage {
-  # Unscaled (original) image width
-  width: Int
-
-  # Unscaled (original) image height
-  height: Int
-
-  # Ask for a version and you might receive it. Versions are not guaranteed to be present.
-  url(version: FeatureImageVersion): String
-}
-
-enum FeatureImageVersion {
-  LARGE_RECTANGLE
-  WIDE
-  SQUARE
-  SOURCE
-}
-
```

Comment:
I didn't wind up using this (though it would be nice to have enums for all the sizes). Just switching over to the generic image type to inherit the crop/resize fields. This winds up not being a breaking change either which is good.